src/components/WebCLI/WebCLI.js

```diff
@@ -68,56 +66,17 @@ const WebCLI = ({ controllerWSHost, credentials, modelUUID, protocol }) => {
       action: "WebCLI command sent",
     });
     inputRef.current.value = ""; // Clear the input after sending the message.
+    setTimeout(() => {
+      // Delay the refresh long enough so that the Juju controller has time to
+      // respond before we request the updated status.
+      refreshModel();
+    }, 500);
```

Comment:
What happens if it takes longer than 500ms to respond?

Reply:
Then it doesn't get reflected until the next poll, which happens every 30s. This is only a temporary fix until we implement the watcher.